Dockerfile

```diff
@@ -3,19 +3,19 @@
 # You can choose to lint this via the following command:
 # docker run --rm -i hadolint/hadolint < Dockerfile
 
-# Since Distroless is Debian based, we take an updated JRE from Zulu's Debian image
-FROM azul/zulu-openjdk-debian:14 AS jre
+FROM alpine:3.12 as install
 
-# Needed for --strip-debug
-RUN apt-get -y update && apt-get --no-install-recommends -y install binutils
+WORKDIR /install
+
+# Install latest JDK 15: we will later use jlink to create a smaller JRE than the default (200MB)
+RUN wget --quiet https://cdn.azul.com/public_keys/alpine-signing@azul.com-5d5dc44c.rsa.pub -P /etc/apk/keys/
+RUN echo https://repos.azul.com/zulu/alpine >> /etc/apk/repositories
+#   binutils is needed for --strip-debug
+RUN apk add --no-cache zulu15-jdk binutils
```

Comment:
Does it work to use their docker image instead of repo fiddling? https://hub.docker.com/r/azul/zulu-openjdk-alpine

Reply:
I hadn't noticed they had a non-jre tag :D didn't scroll far enough. SGTM